test/compliance.node.js

```diff
@@ -11,8 +11,9 @@ describe('compliance', () => {
       let ws = new Ws()
       const addrs = [
         multiaddr('/ip4/127.0.0.1/tcp/9091/ws'),
-        multiaddr('/ip4/127.0.0.1/tcp/9092/ws'),
-        multiaddr('/ip4/127.0.0.1/tcp/9093/ws')
+        multiaddr('/ip4/127.0.0.1/tcp/9092/wss'),
+        multiaddr('/dns4/awesome-dns-server.com/tcp/9092/ws'),
+        multiaddr('/dns4/awesome-dns-server.com/tcp/9092/wss')
```

Comment:
Yep, those should fail... They are being intentionally filtered out, according to the test. 

Not the clearest test case, some refactoring might make sense.